src/wrapper/Websocket.ts

```diff
@@ -185,10 +185,17 @@ export default class Websocket {
                 const baseUrl = (
                     (await core.Supplier.get(this.options.environment)) ?? environments.CartesiaEnvironment.Production
                 ).replace(/^https?:\/\//, "");
-                const params = {
-                    api_key: await core.Supplier.get(this.options.apiKey),
```

Comment:
why are we not using the `core.Supplier.get` pattern? is that auto-generated? just curious

Reply:
Yeah I was just making the expedient change; the `core.Supplier` stuff is an auto-generated tangle.

Reply:
oh, also, the `core.Supplier.get` taps into like a global config, and for this I want it to be only available in ws.connect()